phase1/aws-ec2-terraform/phase2_implementations/simple-weave-single-master.yaml

```diff
@@ -0,0 +1,271 @@
+#cloud-config
```

Comment:
I am not too sure about keeping these as part of the Terraform module, it may be a good idea to externalise these to convey better transparency and phase separation. The Ansible version is included to illustrate how this would work in conjunction with #115, but can be omitted from here.